lib/src/signed_video_h26x_common.c

```diff
@@ -869,12 +852,12 @@ hash_the_hash_list(signed_video_t *self)
 {
   gop_info_t *gop_info = self->gop_info;
   uint8_t *hash = gop_info->hash_of_nalu_hash_list;
-  size_t hash_list_size = gop_info->nalu_list_idx;
-  if (hash_list_size == 0) {
+  if (gop_info->list_idx <= 0) {
```

Comment:
Under what conditions will `list_idx < 0`?
Write a comment on that. Is the action to not do anything and return `SV_OK` appropriate? If not, write a TODO to fix.

Reply:
You still haven't figured out **how** `list_idx` can be negative. Why should we return `SV_INVALID_PARAMETER` under such condition? **If** it is that simple, why not make that change at once instead of adding a TODO for it?